coral-common/src/main/java/com/linkedin/coral/common/transformers/SqlCallTransformer.java

```diff
@@ -49,11 +50,19 @@ public SqlCallTransformer(SqlValidator sqlValidator) {
   protected abstract SqlCall transform(SqlCall sqlCall);
 
   /**
-   * Public entry of the transformer, it returns the result of transformed SqlCall if `predicate(SqlCall)` returns true,
+   * Public entry of the transformer, it returns the result of transformed SqlCall if `condition(SqlCall)` returns true,
    * otherwise returns the input SqlCall without any transformation
    */
   public SqlCall apply(SqlCall sqlCall) {
```

Comment:
Could you add UTs for these 2 changes in `SqlCallTransformer`?

Reply:
[here](https://github.com/linkedin/coral/pull/353/files#diff-dd2d5ae6e8e0d060ead980d488b9b9329bb10599982cfdef69a77432448482dbR135) in this PR is the use of the code change made in apply().
relevant UT - https://github.com/linkedin/coral/blob/cdbdd4ee1a0ec266c234834e141165e0ef1e435c/coral-spark/src/test/java/com/linkedin/coral/spark/CoralSparkTest.java#L823 

Let me know if you meant something else.